CompoundV3-Bots/Governance-Events/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "comp2-2",
+  "version": "0.0.1",
+  "description": "Forta bot that detects some events related to governance actions on Compound v3 Comet contracts",
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/CompoundV3-Bots/Protocol-Reserves",
```

Comment:
Please update the repository field to the correct folder

Reply:
Another good catch, fixed on [aa48e56](https://github.com/NethermindEth/Forta-Agents/pull/371/commits/aa48e5671851f266105a45fea1c639930dbbe716).